pype/plugins/hiero/publish/collect_frames.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+import hiero
+
+from pyblish import api
+
+
+class CollectFrames(api.ContextPlugin):
+    """Collect frames from tags.
+
+    Tag is expected to have metadata:
+    {
+        "family": "frame"
```

Comment:
I would rather use family `image` so we don't need to create new family. Hiero is not using in context `image` family so you will still have the protection from the families blending.

Reply:
When collecting the frame family gets the `image` family as well. That's how its integrated.

I thought `frame` was more intuitive in the context of NukeStudio where the user is interested in publishing a single frame.

Reply:
from what I can see it seem that the final main family will end up being `frame` I agree with @jezscha. It's absolutely ok to have helper families to make it more familiar to the user in their context, but the published main family should be image to keep it consistent.

This this what it visible in Loader at the end of the day, would be `frame`

Reply:
Ahh I see. Yeah, it'll be registered as `frame` atm in the loaders. Will amend.